cmd/dep/testdata/harness_tests/ensure/empty/case1/final/lock.toml

```diff
@@ -0,0 +1,7 @@
+memo = "63510efb9632ec69c1164ce396d7ebea4ad3884b4fa508373da17226d5a39739"
+
+[[projects]]
+  name = "github.com/sdboyer/deptest"
```

Comment:
Hmm...yeah, I hate to bikeshed, but these indents are kinda making me twitch. What's your thinking behind indenting vs. not? Are there any typical standards out there for it? Do we know anything about, say, widespread editor defaults for indentation in TOML files?